packages/canary/src/components/icon.tsx

```diff
@@ -92,6 +92,7 @@ import SshKey from '../icons/ssh-key.svg'
 import GreenCircle from '../icons/green-circle.svg'
 import GreenTick from '../icons/green-tick-nobg.svg'
 import CancelGrey from '../icons/cancel-grey.svg'
+import HarnessLogoText from '../icons/harness-logo-text.svg'
```

Comment:
can we directly import and render this svg? this is technically not an "icon", and keeping it separate will probably be better for the long-term. not a blocker though.

Reply:
agreed, however to run in Playground we'd need to alter the vite config to allow it, I believe.